server/app/controllers/applicant/ProgramSlugHandler.java

```diff
@@ -156,17 +185,31 @@ private Result redirectToOverviewOrReviewPage(
       return Results.badRequest(new ProgramNotFoundException(programSlug).getMessage());
     }
 
+    // For pre-screener forms, redirect to the first block edit page
+    if (activeProgramDefinition.programType().equals(ProgramType.COMMON_INTAKE_FORM)) {
+      return Results.redirect(
+              applicantRoutes.edit(profile, applicantId, activeProgramDefinition.id()))
+          .flashing(FlashKey.REDIRECTED_FROM_PROGRAM_SLUG, programSlug)
```

Comment:
suggestion (non-blocking but highly recommended): add a test case for pre-screeners in `ProgramSlugHandlerTest`